### PR TITLE
Added React Components + eslint (airbnb style)

### PR DESCRIPTION
**Pull request by gorakong**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Header from './Header.jsx';
+import Ratings from './Ratings.jsx';
+import Reviews from './Reviews.jsx';
+
+class App extends React.Component {
+
```
Comment (jcelder) [inline client/src/components/App.jsx:6]: The tabs/spacing on this component looks off. Did you run your entire app through ESLint?

File: client/src/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var Headers = () => {
+
```
Comment (jcelder) [inline client/src/components/Header.jsx:2]: Your component name does not match the file name. You should make sure that they match up.

File: client/src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+}
```
Comment (jcelder) [inline client/src/components/Header.jsx:3]: You are missing a semi-colon at the end of this function declaration statement.

File: client/src/components/Reviews.jsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+var Reviews = () => (
+
+)
```
Comment (jcelder) [inline client/src/components/Reviews.jsx:3]: You are missing a semi-colon at the end of this function declaration statement.

File: server/app.js [hunk 5]
```diff
@@ -5,10 +5,10 @@ var faker = require('faker');
 var mysql = require('mysql');
 var con = require('./database');
 
-app.use(express.static('public'));
+app.use(express.static(__dirname + '/../client/dist'));
```
Comment (jcelder) [inline server/app.js:8]: You should be serving the files from your public folder.

**Review by VALE46GP — APPROVED** [review]
looks good, Gora

File: server/app.js [hunk 6]
```diff
@@ -5,10 +5,10 @@ var faker = require('faker');
...
 app.use(bodyParser.json());
 
-app.get('/', function(req, res) {
+app.get('/seed', function(req, res) {
```
Comment (jcelder) [inline server/app.js:11]: The seeding of your database should not be linked to a public route, and the code should live in a separate file. You will want to seed the database from the command line.

Comment (gorakong) [reply server/app.js:11]: Thanks Josh. I have made the appropriate changes.

**Pull request merged**